local_services/mds_data_generator/mds_test_file_importer.py

```diff
@@ -0,0 +1,349 @@
+                consignment_ref = file.consignment.ConsignmentReference
+
+                try:
+                    s3_key = f"{consignment_ref}/{file_id}"
+                    response = s3.get_object(Bucket=bucket, Key=s3_key)
+                    file_stream = response["Body"].read()
+
+                    file_data = _fetch_file_data(file_id, database_url)
+                    file_data_with_text_content = add_text_content(
+                        file_data, file_stream
+                    )
+
+                    _index_in_opensearch(
+                        file_id,
+                        file_data_with_text_content,
+                        open_search_host_url,
+                        open_search_http_auth,
+                        open_search_ca_certs,
+                    )
+
+                    print(
+                        f"Successfully indexed file: {filename} (ID: {file_id})"
+                    )
+
+                except Exception as e:
+                    print(
+                        f"Error indexing file {filename} (ID: {file_id}): {e}"
+                    )
+
+        except Exception as e:
+            print(f"Error during OpenSearch indexing: {e}")
+
+
+def remove_created_files():
+    """Delete the test files that were created."""
+    print("Deleting created files")
+
+    example_folder = os.path.join(
+        os.getcwd(), "local_services/mds_data_generator/example_files"
+    )
+    print(f"Looking in: {example_folder}")
+    print("Files:", os.listdir(example_folder))
+
+    for filename in os.listdir(example_folder):
+        print(f"Checking: {filename}")
+        if filename.startswith("test_"):
+            file_path = os.path.join(example_folder, filename)
+            if os.path.isfile(file_path):
+                os.remove(file_path)
+                print(f"Deleted: {file_path}")
+
+
+def main():
+    """
+    Generate and process test files for AYR testing using Factory Boy.
+
+    Example usage:
+        python mds_test_file_importer.py --num-pdf 2 --num-png 1
+        if no args are passed, the default will be 1 of each file
+    """
+    parser = argparse.ArgumentParser(
+        description="Generate and process test files for AYR testing"
+    )
+    parser.add_argument(
+        "--num-pdf", type=int, default=0, help="Number of PDF files to create"
+    )
+    parser.add_argument(
+        "--num-png", type=int, default=0, help="Number of PNG files to create"
+    )
+    parser.add_argument(
+        "--num-jpg", type=int, default=0, help="Number of JPG files to create"
+    )
+    parser.add_argument(
+        "--num-tiff", type=int, default=0, help="Number of TIF files to create"
+    )
+
+    args = parser.parse_args()
+
+    file_type_counts = {
+        "pdf": args.num_pdf,
+        "png": args.num_png,
+        "jpg": args.num_jpg,
+        "tiff": args.num_tiff,
+    }
+
+    file_type_counts = {k: v for k, v in file_type_counts.items() if v > 0}
+
+    if not file_type_counts:
```

Comment:
Can't you just set default in `parser.add_argument(` rather than overriding here?